public/app/plugins/datasource/tempo/datasource.ts

```diff
@@ -745,22 +745,43 @@ function serviceMapQuery(request: DataQueryRequest<TempoQuery>, datasourceUid: s
 
       // No handling of multiple targets assume just one. NodeGraph does not support it anyway, but still should be
       // fixed at some point.
-      nodes.refId = request.targets[0].refId;
-      edges.refId = request.targets[0].refId;
+      const { serviceMapIncludeNamespace, refId } = request.targets[0];
+      nodes.refId = refId;
+      edges.refId = refId;
+
+      if (serviceMapIncludeNamespace) {
+        nodes.fields[0].config = getFieldConfig(
+          datasourceUid, // datasourceUid
+          tempoDatasourceUid, // tempoDatasourceUid
+          '__data.fields.title', // targetField
+          '__data.fields[0]', // tempoField
```

Comment:
Can we use other identifier than `fields[0]`? The ordering of the fields isn't really checked or required anywhere.

Reply:
I want to say this can be replaced with `__data.fields.title`, but I'm not 100% sure on how to properly test this (`__data.fields[0]` is what was being used until now)🤔

Reply:
Ah ok didn't notice that. Yeah then leave it for now, not really sure right now what is this for and probably better it stays as it is.